_IBC/osmosis-titan.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "osmosis",
+    "client_id": "07-tendermint-3383",
+    "connection_id": "connection-2838"
+  },
+  "chain_2": {
+    "chain_name": "titannet",
```

Comment:
This file name must match the chain_names, so it must be `osmosis-titannet.json`

Reply:
> This file name must match the chain_names, so it must be `osmosis-titannet.json`

Thank you, modified